tests/unit/test_task_utils.py

```diff
@@ -18,9 +19,20 @@
     dtypes_to_json_task,
     union_dfs_task,
     write_to_json,
+    adls_bulk_upload,
 )
 
 
+class MockClass:
```

Comment:
Is this class only for `adls_bulk_upload` or can it be used for other tests? You have specific arguments there, so maybe change the name of this class to be easier to read and use.

Reply:
done!